apps/admin/backend/src/app.ts

```diff
@@ -161,6 +164,15 @@ function buildApi({
       return auth.unprogramCard(constructAuthMachineState(workspace));
     },
 
+    async writeBallotPackageSignatureFile(input: {
```

Comment:
If ballot package exporting lived entirely on the backend, I wouldn't add a new top-level Grout endpoint for this operation. I'd just update the relevant backend handler. For now though, this was the lowest-lift option.